app/utils/servers.js

```diff
@@ -0,0 +1,21 @@
+export function getServerSpectators(server) {
+    const spectatorsNum = parseInt(server.variables.spectators, 10);
+
+    return !isNaN(spectatorsNum) ? spectatorsNum : 0;
+}
+
+export function getServerPlayersOnly(server) {
+    if (server.variables.players) {
+        const playersNum = parseInt(server.variables.players, 10);
+
+        return !isNaN(playersNum) ? playersNum : 0;
```

Comment:
If this is `NaN` I think it would be better to fall back to `server.players` instead of returning `0`.